nix/docker.nix

```diff
@@ -27,18 +27,22 @@ let
   defaultPort = "8090";
   dataDir = "/data";
 
-  startScript = writeScriptBin "start-cardano-wallet-${backend}" ''
-    #!${runtimeShell}
-    set -euo pipefail
+  startScript =
+    let
+      suffix = if backend == "shelley" then "" else "-" + backend;
+      walletExeName = "cardano-wallet" + suffix;
+    in writeScriptBin "start-cardano-wallet-${backend}" ''
```

Comment:
The name of this start script actually doesn't matter. It could be called globbits and the image would still work.
So you can remove the backend from its name.

Reply:
This I noticed, but in-doubt kept the name as before.

`start-cardano-wallet` could make you believe it was shelley specific, even when the backend is `jormungandr`.

Maybe just `start` or `start-wallet`?

Reply:
I went with `start-wallet` now.